staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -601,6 +614,26 @@ impl VirtioGpu {
 
         resource.shmem_offset = None;
 
+        Ok(OkNoData)
+    }
+    fn update_scanout_resource(
+        &mut self,
+        _scanout_id: u32,
+        scanout_data: Option<VirtioScanoutBlobData>,
+        resource_id: u32,
+    ) -> VirtioGpuResult {
+        // Virtio spec: "The driver can use resource_id = 0 to disable a scanout."
+        if resource_id == 0 {
+            return Ok(OkNoData);
+        }
```

Comment:
This wouldn't actually disable the scanout. Which is fine for now, so maybe add a coment like `// TODO: implement disabling scanout` or maybe even log an error that it is not implemented just so we know we ran into it.